src/einsteinpy/utils/scalar_factor.py

```diff
@@ -31,14 +31,10 @@ def scalar_factor(t, era="md", tuning_param=1.0):
 
     """
     T = t.to(u.s).value
-    if era == "md":
-        return tuning_param * (T ** (2 / 3))
-    elif era == "rd":
-        return tuning_param * (T ** (0.5))
-    elif era == "ded":
-        hubble_const = (constant.Cosmo_Const / 3) ** 0.5
-        val = np.e ** (hubble_const.value * T)
-        return tuning_param * val
+      if era in era_exponents:
```

Comment:
Where is `era_exponents` defined? And this approach would not work with `era == "ded"`. There also seems to be an indentation issue here.